docs/additional-details/migrating-from-react-rails.md

```diff
@@ -1,17 +1,32 @@
 ## Migrate From react-rails
-If you are using [react-rails](https://github.com/reactjs/react-rails) in your project, it is pretty simple to migrate to [react_on_rails](https://github.com/shakacode/react_on_rails).
 
-- Remove the 'react-rails' gem from your Gemfile.
+In this guide, it is assumed that you have upgraded the `react-rails` project to use `shakapacker` version 7. To this end, check out [Shakapacker v7 upgrade guide](https://github.com/shakacode/shakapacker/tree/master/docs/v7_upgrade.md). Upgrading `react-rails` to version 3 can make the migration smoother but it is not required.
```

Comment:
This is a BAD BAD assumption!

Reply:
Then there will be many scenarios. Since ROR 13.4.0 defaults to Shakapacker 7, coming from react-rails < 3 with Webpacker to react_on_rails 13.4 with Shakapacker 7 will have complexities of migrating from Webpacker to Shakapacker. Or let's say from react-rails with Sprockets to react_on_rails Shakapacker 7.

So I thought about doing all the migrations to react-rails 3 + Shakapacker 7 to get rid of all such complications. With that, the migration to react_on_rails will be easy.

This would be the easiest migration guide. Upgrade react-rails project to the latest version and then start the migration.